game/building/src/index.scss

```diff
@@ -0,0 +1,56 @@
+/**
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+@import './widgets/_font.scss';
+
+html, body {
+  overflow: hidden;
+  margin: 0;
+  padding: 0;
+  /* background-image: url('../images/cube-bg.jpg');
```

Comment:
This seems like it can be removed along with the committed image file game/building/src/images/cube-bg.jpg